pkg/controller/replication_controller.go

```diff
@@ -64,6 +64,10 @@ const (
 	// 3000 pods. Just creation is limited to 30qps, and watching happens with
 	// ~10-30s latency/pod at scale.
 	ExpectationsTimeout = 6 * time.Minute
+
+	// An rc is temporarily suspended after creating/deleting these many replicas.
+	// It resumes normal action after observing the watch events for them.
+	BurstReplicas = 500
```

Comment:
Last comment: what do you think about making this a parameter to the NewReplicationManager function?